### ISSUE TITLE
Fix S4790 FN: Support raw string literals

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6298]
### Description

FN for S4790 (CreatingHashAlgorithms).

Using C#11 raw string literals, the rule is not raising an issue.

### Repro steps

```csharp
string part1 = """System.Security.Cryptography""";
string part2 = """SHA1""";

void RawStringLiterals(byte[] temp)
{
    using var SHA1HashAlgorithmWithNamespaceInterpolatedRawStringLiteral = HashAlgorithm.Create($$"""{{part1}}.{{part2}}"""); // FN
}

```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
